x-pack/solutions/observability/plugins/slo/public/pages/slo_management/components/slo_management_search_bar.tsx

```diff
@@ -47,8 +47,8 @@ export function SloManagementSearchBar({ onRefresh }: Props) {
       query={{ query: state.search, language: 'text' }}
       onQuerySubmit={({ query: value }) => {
         onStateChange({ ...state, search: value?.query ? (value?.query as string) : '' });
```

Comment:
Unrelated to your change, but can you cleanup the onStateChange:
```suggestion
       onStateChange({ search: String(value?.query ?? '') });
```